api/src/certification/enrolment/infrastructure/repositories/index.js

```diff
@@ -16,12 +17,13 @@ import * as sessionRepository from './session-repository.js';
  * @typedef {candidateRepository} CandidateRepository
  * @typedef {centerRepository} CenterRepository
  * @typedef {complementaryCertificationRepository} ComplementaryCertificationRepository
- * @typedef {SessionRepository} SessionRepository
- * @typedef {CertificationCandidateRepository} CertificationCandidateRepository
- * @typedef {CertificationCenterRepository} CertificationCenterRepository
- * @typedef {CertificationCpfCountryRepository} CertificationCpfCountryRepository
- * @typedef {CertificationCpfCityRepository} CertificationCpfCityRepository
- * @typedef {SessionForAttendanceSheetRepository} SessionForAttendanceSheetRepository
+ * @typedef {cessionRepository} SessionRepository
```

Comment:
petite typo sur `cessionRepository `

```suggestion
 * @typedef {sessionRepository} SessionRepository

```

Reply:
Deja traité (la typo a ete remonté dans un autre commentaire)